src/pyeconet/equipment/water_heater.py

```diff
@@ -100,7 +100,7 @@ def leak_installed(self) -> bool:
 
     @property
     def has_shutoff_valve(self) -> bool:
-        return self._equipment_info.get("@VALVE") is not None
+        return self._equipment_info.get("@VALVESTATUS", {}).get('title', "").startswith("Shut-OFF Valve - ")
```

Comment:
Do you not have a @VALVE in your response? I'm worried about changing this since I know it was working for my device I'm wondering if we could do an `or` check here

Reply:
Yes, and its values are the same whether the valve is open or closed.
```
                        "@VALVE": {
                            "constraints": {
                                "dialog": [
                                    {
                                        "message": "Closing the shut off valve will disable the water heater",
                                        "title": "Closing Valve",
                                        "value": 0
                                    }
                                ],
                                "error": []
                            },
                            "value": ""
                        },
```

Reply:
Ahh okay then I don't think you need this change. This just indicates if your unti has a shutoff valve or not. Units without one won't have that block at all.

Reply:
I noticed that this response in the examples section shows the same `@VALVE` value, but also underneath in the `@VALVESTATUS` section says `"Shut-OFF Valve not installed"`. Perhaps it's an incorrect assumption, but that leads me to believe that `@VALVE` can't be trusted for that purpose.

 https://github.com/w1ll1am23/pyeconet/blob/master/src/pyeconet/example_responses/get_locations_water_heater_electric_gas_mode.json#L253

Reply:
Well shoot, okay.